src/log-tlslog.c

```diff
@@ -307,10 +308,17 @@ static int LogTlsLogger(ThreadVars *tv, void *thread_data, const Packet *p,
 
     MemBufferReset(aft->buffer);
     MemBufferWriteString(aft->buffer,
-                         "%s %s:%d -> %s:%d  TLS: Subject='%s' Issuerdn='%s'",
-                         timebuf, srcip, sp, dstip, dp,
-                         ssl_state->server_connp.cert0_subject,
-                         ssl_state->server_connp.cert0_issuerdn);
+                         "%s %s:%d -> %s:%d  TLS: ",
+                         timebuf, srcip, sp, dstip, dp);
+
+    if ((ssl_state->flags & SSL_AL_FLAG_SESSION_RESUMED) != 0) {
+        MemBufferWriteString(aft->buffer, "Session resumed");
+    } else {
+        MemBufferWriteString(aft->buffer,
+                             "Subject='%s' Issuerdn='%s'",
+                             ssl_state->server_connp.cert0_subject,
+                             ssl_state->server_connp.cert0_issuerdn);
```

Comment:
Instead of this if-else, I think you should do individual if checks to see if subject and issuer is set and print if they are, like is done with the other fields in LogTlsLogExtended().